src/sidebar/components/HelpPanel.tsx

```diff
@@ -44,13 +50,15 @@ type HelpPanelTabProps = {
  */
 function HelpPanelTab({ linkText, url }: HelpPanelTabProps) {
   return (
-    <div className="flex-1 border-r last-of-type:border-r-0">
-      <Link
-        href={url}
-        classes="flex items-center justify-center space-x-2 text-color-text-light text-lg font-medium"
-        target="_blank"
-      >
-        <span>{linkText}</span> <Icon name="external" classes="w-3 h-3" />
+    <div
+      // Set this element's flex-basis per its flex parent and also establish
```

Comment:
I'm not sure what is meant by "per its flex parent" here. `flex-1` sets the `flex-basis` to `0%`, or per the Tailwind docs "Use flex-1 to allow a flex item to grow and shrink as needed, ignoring its initial size".

Reply:
Ech, poorly-written comment on my end. I'll adjust.